### DISCUSSION TITLE
Users should be able to pass metadata and tags via `overrideConfig` when using the Flowise integration

### DISCUSSION THREAD

**Discussion by giorgosera** [langfuse/langfuse#999] [Ideas]
I'd like to be able to pass metadata or at least tags when using Langfuse via Flowise. 

It seems that this is not possible at the moment. All the examples, demonstrate how to pass `userId` but there is not really any documentation on how (and if) this is possible. 

Here's what I tried but did not work.

```
{
    "question": "hi there",
    "overrideConfig": {
        "analytics": {
            "langFuse": {
                "userId": "user1",
                "metadata": {
                  "foo": "bar"
                },
                "tags": ["foo"]
            }
        }
    }
}
```

I tried to find the place in the code where Flowise sets up the trace but I got confused :) It seems that there are multiple places where the setup is happening afaict. Also, please note that I am not using OpenAI.

Thanks 🙏

**Comment by marcklingen** [comment]
Hi, just added this to the Langchain JS integration and made a PR to upgrade to the latest version in Flowise. This should be solved once the PR is merged: https://github.com/FlowiseAI/Flowise/pull/2055/

**Reply by phmalek** [reply]
I tried it using a free account and it worked. 
I had the same issue when I wanted to use local container of weaviate on Flowise. Instead of http://localhost:$PORT I tried $container_name:$Port and it worked. I used the same strategy for langfuse, but it didn't work.
